src/semeio/forward_models/__init__.py

```diff
@@ -0,0 +1,160 @@
+# class notation
+from ert.config.forward_model_step import ForwardModelStepJSON, ForwardModelStep
+
+
+class Design2Params(ForwardModelStep):
+    def __init__(self):
+        super().__init__(
+            name="DESIGN2PARAMS",
+            executable="design2params",
+            arglist=["<IENS>", "<xls_filename>", "<designsheet>", "<defaultssheet>"],
```

Comment:
We should probably use python types here

```
arglist[("<IENS", int), ("<xls_filename>", Annotated[Path, PathType('file')]) ...
```

That way we can use standard functions for checking types. Not sure exactly which one, but `beartypes` should have something for this.

Reply:
I think we can leave types/validation up to "users" implementing this for a simpler interface on our end